pkg/pipelines/bootstrap.go

```diff
@@ -140,18 +142,19 @@ func Bootstrap(o *BootstrapParameters) error {
 
 	//  Create Role, Role Bindings, and ClusterRole Bindings
 	outputs = append(outputs, createRoleBindings(namespaces, sa)...)
+	outputs = append(outputs, statustracker.Resources(namespaces["cicd"])...)
```

Comment:
Status tracker resource should not be added if status-tracker-token is not provided.    

Also, the status-tracker-token option sets up the "github-auth" secret.   What is "github-auth" consumed by status tracker?  We should rename the field "GitHubToken" to something like "StatusTrackerToken" to reflect their relationship.